test/ci-test-weather.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Regarding the tests files, to keep consistence and/or ease maintenance, we should either mirror the folder structure of the models within the test folder (if you want to keep the test separate from the models), or create a subfolder in each model folder called test (sister to doc). I personally prefer the latest, so that definition and test remain close to each other.